radio/src/translations/jp.h

```diff
@@ -495,16 +495,16 @@
 #define TR_NODATA                      CENTER "データなし"
 #define TR_US                          "us"
 #define TR_HZ                          "Hz"
-#define TR_TMIXMAXMS                   "Tmix最大値"
+#define TR_TMIXMAXMS                   "Tmix max"
 #define TR_FREE_STACK                  "Free stack"
 #define TR_INT_GPS_LABEL               "内蔵GPS"
-#define TR_HEARTBEAT_LABEL             "ハートビート"
-#define TR_LUA_SCRIPTS_LABEL           "LUAスクリプト"
-#define TR_FREE_MEM_LABEL              "空きメモリ"
-#define TR_TIMER_LABEL                 "タイマー"
-#define TR_THROTTLE_PERCENT_LABEL      "スロットル %"
-#define TR_BATT_LABEL                  "バッテリー"
-#define TR_SESSION                     "セッション"
+#define TR_HEARTBEAT_LABEL             "Heartbeat"
+#define TR_LUA_SCRIPTS_LABEL           "Lua scripts"
+#define TR_FREE_MEM_LABEL              "Free mem"
+#define TR_TIMER_LABEL                 "Timer"
+#define TR_THROTTLE_PERCENT_LABEL      "Throttle %"
+#define TR_BATT_LABEL                  "Battery"
+#define TR_SESSION                     "Session"
```

Comment:
Were changes like this intentional (reverting JP to EN)? Just checking ;)

Reply:
It is intentional.
We dared to use English because if we used Japanese on the debug screen, the line break point would be off and the strings would overlap.